web-app/js/portal/filter/BooleanFilterPanel.js

```diff
@@ -83,6 +82,10 @@ Portal.filter.BooleanFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
         this.checkbox.setValue(false);
     },
 
+    needsFilterRange: function() {
+        return false;
+    },
+
     _setExistingFilters: function() {
         this.re = new RegExp(this.filter.name + " = (.*?)( |$)");
```

Comment:
This is old code for parsing values out of CQL. This will be killed with the advent of the new Filter model.